index.ts

```diff
@@ -1,53 +1,28 @@
+// index.ts
+
 import express, { Application } from "express";
-import mongoose from "mongoose";
 import dotenv from "dotenv";
-// import authRouter from "./routes/auth";
-// import userRouter from "./routes/user";
-// import orderRouter from "./routes/order";
-// import cartRouter from "./routes/cart";
-// import stripeRouter from "./routes/stripe";
-// import productRouter from "./routes/product";
 import cors from "cors";
-// import mysql from "mysql";
+import database from './src/config/mySql';
 
-dotenv.config();
+dotenv.config({ path: `.env.${process.env.NODE_ENV}` }); 
 
 const app: Application = express();
 
 app.use(cors());
 app.use(express.json());
 
-// MongoDB Connection
-mongoose
```

Comment:
move this commented code to mongoDB.ts for refernce

Reply:
ok, sure